src/quo2/components/tags/tag.cljs

```diff
@@ -59,22 +59,36 @@
       label])])
 
 (defn tag
+  "opts
+   {:type         :icon/:emoji/:label
+    :size         32/24
+    :on-press     fn
+    :blurred?     true/false 
+    :resource     icon/image
+    :labelled?    true/false
+    :disabled?    true/false}
+  
+   opts
+    - `blurred`  boolean: use to determine border color if the background is blurred
+    - `type`     can be icon or emoji with or without a tag label
+    - `labelled` boolean: is true if tag has label else false"
   [_ _]
-  (fn [{:keys [id on-press disabled size resource active accessibility-label
-               label type labelled blurred icon-color] :or   {size 32}}]
-    (let [state (cond disabled :disabled active :active :else :default)
+  (fn [{:keys [id on-press disabled? size resource active accessibility-label
```

Comment:
so you have adjusted a lot of props here. Can you make sure that places where this component are used across the codebase are also updated to this api 🙏